edubots_surveys.Rnw

```diff
@@ -1468,35 +1468,31 @@ sciences show some significant differences. We do not have any kind of explanati
 this behavior, other than technologies are perceived, and adopted, in a
 different way depending on background and, apparently, gender.
 
-One of the most interesting questions that can arise from the analysis of the
+One of the most interesting questions arising from the analysis of the
 results, and that should require further study, is: Are teachers really aware
 of the possibilities of chatbots in the classroom? The survey
-only listed a few functionalities for the chatbots and just a few subjects selected
-the option "Others". This may suggest  that most teachers did not really understand
-at this point the actual potential of chatbots, since the responses listed were by no means exhaustive, and if the responders were more acquainted with the possibilities of the technology, possibly this response would have shown up many more times. Another way to look about it is
-that, unless you provide the specific instructions on how to use a
-tool, the teaching staff is already too busy or burn out to think about
-innovation.
-
-We can speculate that their opinions towards chatbots are informed by negative
-experiences with customer support chatbots, which is what most people will have
-experienced. This will probably require further study, but also will need to be
-taken into account when introducing chatbot technologies in the classroom:
-it will have to be as simple, and as satisfactory, as possible,
-to (possibly) change this existing perception.
+only listed a few functionalities for the chatbots, and just a few subjects selected
+the option "Others". This may suggest that most teachers did not really understand
+at this point the actual potential of chatbots, since the responses listed were by no means exhaustive. If the responders were more acquainted with the possibilities of this technology, this response would possibly have shown up many more times. Another way to look at it is that, unless you provide the specific instructions on how to use a
+tool, the teaching staff is already too busy or burnt out to think about innovation.
+
+We can speculate that teachers' opinions of chatbots are influenced by negative
+experiences with customer support chatbots, which is what most people are likely to have
+experienced. Further study may be needed to confirm this, but it is important to 
+keep in mind when introducing chatbot technologies in the classroom.
+To potentially change this existing perception, it will be necessary to make the chatbot as simple and satisfying to use as possible.
```

Comment:
to make chatbots as simple

Reply:
```suggestion
To potentially change this existing perception, it will be necessary to make chatbots as simple and satisfying to use as possible.
```